resources/resinos.robot

```diff
@@ -40,6 +40,23 @@ File list "@{files_list}" does not exist in "${mount_destination}"
     : FOR   ${files_line}     IN  @{files_list}
     \   File Should Not Exist  ${mount_destination}${files_line}  msg="Backup file ${files_line} found on the rootfs that should not exist." 
     
+Clone "${git_url}" in "${dir}"
+    ${result} =  Run Process    git clone ${git_url}    shell=yes   cwd=${dir}
+    Process ${result}
+
+Set up getty service for "${device_type}"
+    ${result} =  Run Process    ./serial-it.sh --root-mountpoint ${mount_destination} -b ${device_type}     shell=yes   cwd=${tmp_dir}/serial-it
+    Process ${result}
+
+Run minicom "${script}" at startup and capture "${output}"
+    Run Process    minicom -D unix\#console.sock -S ${script} -C ${output}    shell=yes   cwd=/tmp    timeout=1s
+    File Should Not Be Empty    ${output}
+
+Check if service is active from "${output}"
```

Comment:
This keyword seems wrong - maybe a master keyboard that does not use minicom scripts but takes in the service name from the Keyword like so would be nice

Check if service "resin-info@tty1.service" is running using socket "/tmp/console.sock" is a desirable keyword without all the extra noise.